File: android/snag/src/main/java/com/snag/models/SnagAppInfo.kt
package com.snag.models

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
data class SnagAppInfo(
    @SerialName("bundleId")
    val bundleId: String? = null,
    @SerialName("isReactNative")
    val isReactNative: Boolean = false
)

File: ios/Snag/Models/SnagAppInfo.swift
import Foundation

public struct SnagAppInfo: Codable, Sendable {
    public var bundleId: String?
    public var isReactNative: Bool = false
    
    public init(bundleId: String? = nil, isReactNative: Bool = false) {
        self.bundleId = bundleId
        self.isReactNative = isReactNative
    }
}
